clients/python/src/osparc/_api_files_api.py

```diff
@@ -251,11 +251,15 @@ def _search_files(
         sha256_checksum: Optional[str] = None,
         timeout_seconds: int = DEFAULT_TIMEOUT_SECONDS,
     ) -> PaginationGenerator:
+        kwargs = {
+            "file_id": file_id,
+            "sha256_checksum": sha256_checksum,
+            "timeout_seconds": timeout_seconds,
+        }
+
         def _pagination_method():
             return super(FilesApi, self).search_files_page(
-                file_id=file_id,
-                sha256_checksum=sha256_checksum,
-                _request_timeout=timeout_seconds,
+                {k: v for k, v in kwargs.items() if v is not None}
```

Comment:
dont you need to expand them? i.e. add `**`?

Reply:
If so, how come your tests did not detect that? TIP: Add tests that fail and then provide a fix.